### ISSUE TITLE
Google chrome NVDA faille to read history content

### ISSUE THREAD

**Issue by ahmedabdelslam** [nvaccess/nvda#14655]
Hello, 
NVDA faille to read history content in google chrome.
 this issue also happens in some web pages has tab structure but 
I face this issue in internal system so that is hard to reproduce I think google chrome history have the same problem 

for example, when select specific tab then press h to go to tab content NVDA speak no next heading like history dialog but you can navigate by tab key then NVDA will read the content

### Steps to reproduce:

open google chrome 
press control + h 
press escape to exit focus mode then press h 
### Actual behavior:

NVDA speaks no next heading

### Expected behavior:

NVDA should speak last history date at heading level two 

#### NVDA installed/portable/running from source:
NVDA installed/portable
#### NVDA version:
NVDA 2022.4 / latest master
#### Windows version:
Windows 11 Pro 22H2 build 22621.1265
#### Name and version of other software in use when reproducing the issue:
Google Chrome version 110.0.5481.100
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
the same behavior in some older versions
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by Adriani90** [comment]
This is because the system focus lands on the first checkbox entry in the list below the heading which is at the top of the dialog when entering the list. If you move to browse mode, the virtual cursor will still focus the checkbox. If you press shift+h, NVDA will report the heading. This behavior is expected and makes sense, because the system focus lands on an item with an interaction as defined by the web author. This is also defined by the tab index.

Note that in focus mode, NVDA reports the heading and the first checkbox together, so you know when entering a new history date when pressing up or down arrow.

I don't see an accessibility issue here, so I am closing this issue as works for me. Please feel free to comment in case I am wrong and we can reopen it.

**Comment by ahmedabdelslam** [comment]
Hello Adriani,

I think NVDA should able to navigate heading or checkboxes ones getting browse mode.
 we have more checkboxes after the focus .

Thank you

**Comment by Adriani90** [comment]
@ahmedabdelslam you can navigate with h between headings, when there are more than one heading. Otherwise NVDA focuses the first checkbox under the only heading which is in your dialog and that
s why NVDA says, no next heading. If you press shift+h you navigate to the only heading in the dialog.

You can navigate the checkboxes in browse mode with the key x.

**Comment by ahmedabdelslam** [comment]
@ahmedabdelslam 
I try this with other screen reader and it works fine 
maybe I can't explain it well
@josephsl 
would you take a look at that
